core/vmpu/src/mpu_armv8m/vmpu_armv8m_mpu.c

```diff
@@ -283,6 +284,14 @@ bool vmpu_buffer_access_is_ok(int box_id, const void * addr, size_t size)
         return false;
     }
 
+    /* Check if addr range lies in page heap. */
```

Comment:
This should be in its own commit, as it fixes the v8-M implementation.

"armv8m: Check page heap in vmpu_buffer_access_is_ok"